cli.js

```diff
@@ -80,7 +80,7 @@ var cli = meow([
 	'  ava --init foo.js',
 	'',
 	'Default patterns when no arguments:',
-	'test.js test-*.js test/**/*.js'
+	'test.js test-*.js test/**/*.js __tests__/**/*.js **/*.test.js'
```

Comment:
Updated. :+1: I've subscribed to #713. As soon as that gets merged, I'll give this another look.